src/main/java/seedu/staffsnap/model/applicant/Descriptor.java

```diff
@@ -0,0 +1,28 @@
+package seedu.staffsnap.model.applicant;
+
+/**
+ * Enum for valid Descriptors
+ */
+public enum Descriptor {
```

Comment:
I like the choice to set this up as an enum, since it will be immutable 😄

Reply:
Thank you Craig!